### ISSUE TITLE
Create basic ReadMe template

### ISSUE THREAD

**Issue by rimutaka** [thegooddocsproject/templates#110]
ReadMe is the default document displayed on the homepage of the project and is often the only document for the repo. We should create several ReadMe templates for different project types starting with the most basic one.

Creating one from scratch is of limited value because it would have to be tested by large number of readers before we can say if it is a good or bad example. Let's re-use what's already out there and has been proven to work.

**Steps:**

1. Collect examples of GOOD ReadMe files.
2. Collect examples of BAD ReadMe files.
3. Deconstruct
4. Create a composite ReadMe that includes WHAT, WHY and HOW.

#### What's GOOD, what's BAD?
It can be a well illustrated and grammatically correct doc, but still unhelpful. Another doc may be very short, but is all what's needed to understand and start using the code.

#### BAD examples
We can't just single out someone's readme as bad. It would be good to have examples where a bad version was turned into a good version.

#### Target project type

a) a small team or a solo dev
b) ReadMe is the only doc in the repo
c) the code does only one or two things, but does them well
d) ReadMe is the only or primary source of info about the project

#### Useful references
* https://github.com/kylelobo/The-Documentation-Compendium/tree/master/en/README_TEMPLATES
* https://docs.google.com/document/d/1sPaVeFQFEqUs0w0sbvvWz1yW3rZd7Lba2_hNheQG25Y/edit

**Comment by camerons** [comment]
#152 was closed as a duplicate of this.

cwcromwell commented 
Projects on GitLab and GitHub need a readme with information such as:

-- how to contribute to a project
-- project architecture
-- how to build the project locally (including depencies that you might have to install)
-- link to more documentation if it exists.

There might be more needed in this outline.

@mjang
 
mjang commented 
GitLab template example for GraphQL: https://docs.gitlab.com/ee/development/documentation/styleguide.html#graphql-api

@ScriptAutomate
 
ScriptAutomate commented 

This would be excellent for The Good Docs Project repositories to reference, too. TGD has a repository that is missing a README, and could make use of it!

The Good Docs Project website source code
Potential reference material for readme templating: https://github.com/matiassingers/awesome-readme

@Loquacity
 
Member
Loquacity commented 
Possible duplicate? #110

@mjang
 
mjang commented 
You're welcome to assign me an issue, @ScriptAutomate

@ScriptAutomate
 
ScriptAutomate commented 
@Loquacity Agreed.

@cwcromwell I think this issue should be closed as a duplicate of #110

**Comment by ScriptAutomate** [comment]
Just going to leave this here, too: https://github.com/thegooddocsproject/thegooddocsproject.github.io/issues/33
